app/javascript/orangelight/availability.es6

```diff
@@ -65,7 +67,10 @@ export default class AvailabilityUpdater {
           });
 
       } else {
-        url = `${this.bibdata_base_url}/bibliographic/${this.id}/availability.json`;
```

Comment:
I'm curious -- does this leave that endpoint unused?

Reply:
Hum...it might, at least from Orangelight.

Reply:
Never mind, we still use it when we make the call for `deep` info: https://github.com/pulibrary/orangelight/blob/main/app/javascript/orangelight/availability.es6#L136-L141

Reply:
oh, great, thanks for this follow-up